### PR TITLE
Hungarian Language Added

### PR DESCRIPTION
**Pull request by Gegenter**

### CODE REVIEW

File: base.hu.i18n [hunk 1]
```diff
@@ -0,0 +1,2 @@
+RpnApp = "RPN"
+RpnAppCapital = "RPN"
```
Comment (boricj) [inline base.hu.i18n]: Newline missing at end of file.

**Review by boricj — APPROVED** [review]
Looks good, but you should probably contribute this to upstream first.

**Comment by quentinguidee** [comment]
@boricj I don't think the Hungarian language must be pushed upstream, since Epsilon is not compatible with Hungarian (but it's up to you)

**Comment by Gegenter** [comment]
Just a side note, for RPN it doesn't translate anything, since there is nothing really to translate. So yeah just a side note.

**Comment by quentinguidee** [comment]
So @boricj I merged it, but you can pull this PR to your fork if you want ;)

**Pull request merged**